larq/quantizers.py

```diff
@@ -136,7 +136,7 @@ class SteTern:
     \\[
     q(x) = \begin{cases}
     +1 & x > \Delta \\\
-    0 & |x| > \Delta \\\
+    0 & |x| < \Delta \\\
     -1 & x < \Delta
```

Comment:
This line needs to be changed too, correct?
```suggestion
     -1 & x < - \Delta
```